frontend/src/metabase/plugins/index.ts

```diff
@@ -276,14 +280,16 @@ export const PLUGIN_MODERATION = {
 };
 
 export const PLUGIN_CACHING = {
+  cacheTTLFormField: null as any,
   dashboardCacheTTLFormField: null,
   questionCacheTTLFormField: null,
-  getQuestionsImplicitCacheTTL: (_question?: any) => null,
-  QuestionCacheSection: PluginPlaceholder,
-  DashboardCacheSection: PluginPlaceholder,
-  DatabaseCacheTimeField: PluginPlaceholder,
```

Comment:
Did you forget to add a type for your plugin?

Reply:
I tried to add types to these plugins in a separate branch, but there were some type inconsistencies in preexisting code that made this hard, so I decided to pull back.